js/charts/scatter.js

```diff
@@ -128,13 +160,13 @@ function scatter() {
 
             lasso.notSelectedItems().selectAll('rect');
 
-            var confirm = $(scope).parent().find('div.confirm')
-                .css('visibility', 'visible');
+            var confirm = d3.select('.confirm')
```

Comment:
this can be conflicting. As in flair-bi there can be multiple widgets running parallel and the confirm button could be in any widget. If we only do d3.select('.confirm') then it can select any confirm button and not of this specific chart.